modules/overview-of-accelerators.adoc

```diff
@@ -16,10 +16,10 @@ If you work with large data sets, you can use accelerators to optimize the perfo
 * NVIDIA graphics processing units (GPUs)
 ** To use compute-heavy workloads in your models, you can enable NVIDIA graphics processing units (GPUs) in {productname-short}. 
 **   To enable GPUs on OpenShift, you must install the link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator].
-* Habana Gaudi devices (HPUs)
-** Habana, an Intel company, provides hardware accelerators intended for deep learning workloads. You can use the Habana libraries and software associated with Habana Gaudi devices available from your notebook.
-** Before you can enable Habana Gaudi devices in {productname-short}, you must install the necessary dependencies and the version of the HabanaAI Operator that matches the Habana version of the HabanaAI workbench image in your deployment. For more information about how to enable your OpenShift environment for Habana Gaudi devices, see link:https://docs.habana.ai/en/v1.10.0/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator v1.10 for OpenShift] and link:https://docs.habana.ai/en/v1.13.0/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator v1.13 for OpenShift].
-** You can enable Habana Gaudi devices on-premises or with AWS DL1 compute nodes on an AWS instance.
+* Intel Gaudi AI accelerators (HPUs)
+** Habana, an Intel company, provides hardware accelerators intended for deep learning workloads. You can use the Habana libraries and software associated with Intel Gaudi AI accelerators available from your notebook.
```

Comment:
If you go to the official Habana website: https://habana.ai/, you can see that the word Habana has been removed entirely. Only the URL mentions Habana. It might be the case that the company called Habana which Intel acquired as been dissolved, and its corporate structure merged in with Intel. Therefore there could no longer be a company called "Habana". Perhaps we should consider changing this.

Reply:
Fair, have updated.